namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-events-prod/resources/topic.tf

```diff
@@ -68,3 +68,24 @@ resource "kubernetes_secret" "prison_data_compliance" {
     topic_arn         = module.offender_events.topic_arn
   }
 }
+
+module "offender_assessments_events" {
+  source             = "github.com/ministryofjustice/cloud-platform-terraform-sns-topic?ref=4.1"
+  team_name          = var.team_name
+  topic_display_name = "offender-assessments-events"
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "offender_assessments_events" {
+  metadata {
+    name      = "offender-assessments-events-topic"
+    namespace = var.namespace
```

Comment:
Namespace looks wrong, this should be the assessment namespace - or is that not running in CP?

Reply:
not running in CP, so it will need manual extraction as per slack thread